data/korath/korath.txt

```diff
@@ -356,6 +356,16 @@ fleet "Korath Surveillance"
 		"Ra'at Ik 621 (Probes)"
 		"Kas'ik Tek 7" 9
 
+fleet "Korath Stealth Relay"
+	government "Korath"
+	names "korath"
+	fighters "korath probes"
+	cargo 0
```

Comment:
Just wondering, do the drones even have cargo space? Also mb add uninterested to their personality
I like that they self destruct

Reply:
Yes, 4 cargo space, and they are now uninterested.

Reply:
Yes, 4 cargo space, and they are now uninterested.